cmd/controller-manager/app/options/options.go

```diff
@@ -50,11 +51,31 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.LimitedScope, "limited-scope", false, "Whether the federation namespace will be the only target for federation.")
 	fs.DurationVar(&o.ClusterMonitorPeriod, "cluster-monitor-period", time.Second*40, "How often to monitor the cluster health")
+
+	fs.BoolVar(&o.LeaderElection.LeaderElect, "leader-elect", true, "Start a leader election client and gain leadership before "+
+		"executing the main loop. Enable this when running replicated controller-manager for high availability.")
+	fs.DurationVar(&o.LeaderElection.LeaseDuration, "leader-elect-lease-duration", util.DefaultLeaderElectionLeaseDuration, ""+
```

Comment:
(No action required) I'm assuming you copied this from somewhere, but I'd rather not use string concatenation to define multiline strings if at all possible.  Editing them is not fun.